### PR TITLE
Yara rules

### PR DESCRIPTION
**Pull request by brifordwylie**

### CODE REVIEW

**Comment by jeffbaumes** [comment]
This errors for me:

```
python yara_matches.py -y ../brothon/utils/yara_rules/ -e .

Loading Yara Rules from ../brothon/utils/yara_rules/
Traceback (most recent call last):
  File "yara_matches.py", line 49, in <module>
    my_rules = yara_rules.YaraRules(rule_path=args.yara_dir)
  File "/home/jeff/code/BroThon/brothon/utils/yara_rules.py", line 29, in __init__
    self.yara_rules = yara.compile(self.rule_path)
yara.SyntaxError: ../brothon/utils/yara_rules/(1): input in flex scanner failed
```

Something to do with certain yara versions not supporting directories? I had  done `pip install yara-python` beforehand.

https://groups.google.com/forum/#!topic/yara-project/sn-fLtu5rWs

**Comment by brifordwylie** [comment]
@jeffbaumes hey someone's actually using this :) . Yep.. if you look at the example I'm now pointing it to a specific yara file.. but I see that I need to change the docstrings and should make this more explicit. Thanks @jeffbaumes

**Comment by brifordwylie** [comment]
In fact.. I'm going to remove all but one of yara rules and put in a README.md that shows people how to use/download the full set of rules: https://github.com/Yara-Rules/rules rules

File: examples/yara_matches.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        print('Unrecognized args: %s' % commands)
+        sys.exit(1)
+
+    # If no args just call help
+    if len(sys.argv) == 1:
+        parser.print_help()
+        sys.exit(1)
+
+    # Sanity check that the args exist and are directories
+    if not os.path.isdir(args.yara_dir):
```
Comment (jeffbaumes) [inline examples/yara_matches.py:40]: Looks like you will fail if it is not a dir - you would prob need to change this check?

**Comment by brifordwylie** [comment]
Made a new PR for this https://github.com/Kitware/BroThon/pull/2

**Pull request closed without merging**